### PR TITLE
Massive backend and backend connection and UI refactor

### PR DESCRIPTION
**Pull request by codecreativity**
My notes

-> After this refactor you will have to re-import the project in android studio
-> Afer this re-install app on dev phone

### CODE REVIEW

**Comment by LoLei** [comment]
Navigation drawer without map working :white_check_mark:

File: app/src/main/java/com/sw_ss16/lc_app/content/LearningCenterContent.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    // -------------------------------
+    // Methods
+    // -------------------------------
+
+    //
+    public void setApplicationContext(Context app_context){
+        this.app_context = app_context;
+    }
+
+    public boolean getLearningCeterFavoriteStatus(int lc_id){
```
Comment (LoLei) [inline app/src/main/java/com/sw_ss16/lc_app/content/LearningCenterContent.java:35]: Typo in method name

**Comment by LoLei** [comment]
Working well and much clearer. :white_check_mark:

File: app/src/main/java/com/sw_ss16/lc_app/content/LearningCenterContent.java (hunk 1 above)
Comment (codecreativity) [inline app/src/main/java/com/sw_ss16/lc_app/content/LearningCenterContent.java:35]: will change this in later PR

**Pull request merged**